app/javascript/components/session-comment-form.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import type { SessionComment } from 'api/session-comments';
+import type { FeedbackSession } from 'api/feedback-sessions';
+import { reactive } from 'vue';
+
+interface Props {
+  session: FeedbackSession
+  loading: boolean
+}
+
+const props = defineProps<Props>();
+
+const form: Partial<SessionComment> = reactive({
+  body: '',
+  feedbackSessionId: props.session.id,
+});
+
+const emit = defineEmits<{
+  (event: 'submit', value: SessionComment): void;
+}>();
+
+</script>
+<template>
+  <form
+    class="flex w-80 flex-col items-start gap-5 rounded-md bg-blue-100 p-4 md:w-auto md:max-w-3xl md:rounded-lg"
+    @submit.prevent="emit('submit', form)"
```

Comment:
🚫 **[tsc]** <2345> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument of type 'Partial<SessionComment>' is not assignable to parameter of type 'SessionComment'.
Types of property 'id' are incompatible.
Type 'number | undefined' is not assignable to type 'number'.
Type 'undefined' is not assignable to type 'number'.

Reply:
@gmq debería dejarlo así?

Reply:
No, en la linea 19 hay que actualizar el evento para que acepte el SessionComment parcial, y también en la linea 18 de `session-comment.vue`.